contrib/tor/monero-over-tor.sh

```diff
@@ -67,7 +77,7 @@ fi
 echo "Starting monerod..."
 HOSTNAME=$(cat "$HOSTNAMEFILE")
 "$monerod" \
-  --anonymous-inbound "$HOSTNAME":18083,127.0.0.1:18083,25 --tx-proxy tor,127.0.0.1:9050,10 \
+  --anonymous-inbound "$HOSTNAME":18083,127.0.0.1:18083,25 --proxy 127.0.0.1:9050 --tx-proxy tor,127.0.0.1:9052,10 \
   --add-priority-node zbjkbsxc5munw3qusl7j2hpcmikhqocdf4pqhnhtpzw5nt5jrmofptid.onion:18083 \
   --add-priority-node 2xmrnode5itf65lz.onion:18083 \
```

Comment:
Remove

Reply:
Some seedodes are good to bootstrap the network. If a contributor could add some trusted nodes

Reply:
Wrong. There are seednodes built in.
please remove those lines. Thx